DESIGNREFERENCE.md

```diff
@@ -0,0 +1,12 @@
+# Project-Tactics use case
+As a player I download the entire repository, create my own battles with my own enemies and maps and *share* this new battles with other players. So other players can play new battles, new characters, new jobs and much more. 
+
```

Comment:
I believe this souds better:
_As a player I can download **Open Tactica** to dive into custom battles reminiscent of classic turn-based strategy JRPGs like Final Fantasy Tactics. I can create and share my own battles and events with other players and enjoy deep customization of characters, skills and classes._